File: apps/institution-api/Models/EntityFramework/PmpLookup.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class PmpLookup
    {
        public string Code { get; set; }
        public string Type { get; set; }
        public string Name { get; set; }
        public int DisplayOrder { get; set; }
    }
}

File: libs/common/models/pmp-lookup.ts



export class PmpLookup {
    public code: string = '';
    public type: string = '';
    public name: string = '';
    public displayOrder: number = 0;

    constructor(init? : Partial<PmpLookup>) {
        
        Object.assign(this, init);
    }
}

